Hotfix/validator manager

cowboy-bebug opened the pull request:
Sorry guys I pushed buggy code (from this https://github.com/plugblockchain/plug-blockchain/commit/dbcbd2a7bef857c5eeb24a1774fae64f565fb8e7). Checked with
```console
$ cargo check -p prml-validator-manager && cargo test -p prml-validator-manager
```

### First commit resolves this issue:
![Screen Shot 2020-07-23 at 4 29 38 PM](https://user-images.githubusercontent.com/6591791/88252358-fce8d480-cd01-11ea-8d43-03f7afa5c7a0.png)

### Second commit resolves this issue:
![Screen Shot 2020-07-23 at 4 29 20 PM](https://user-images.githubusercontent.com/6591791/88252361-ffe3c500-cd01-11ea-8392-c009cd26e3ec.png)

cowboy-bebug committed:
Make pallet-session std compatible

cowboy-bebug committed:
Ignore code block (no_run compiles)

jordy25519 commented:
Thanks, how did this pass CI before though?
Do we need to add a new CI step to support this module?

cowboy-bebug commented:
Yeah let me investigate that and get back to you.

cowboy-bebug committed:
Add multiline scalar

cowboy-bebug commented on .circleci/config.yml:
```diff
@@ -60,7 +60,7 @@ commands:
     steps:
       - run:
-          command:
+          command: |
```
This was the culprit. The pipeline was running `cargo --version cargo test --release --all` in a single line so `cargo test` part never executed.

cowboy-bebug replied:
As for the doc tests, `cargo test --all` doesn't test docs and we have to run `cargo test --doc` separately.
But `cargo test -p some-crate` tests for that crate's doc (little bit confusing) <- and this is how the doc error was caught.

cowboy-bebug committed:
Sneak in small improvement
Apparently --all is deprecated alias for --workspace

cowboy-bebug committed:
Fix ui tests (stderr)

cowboy-bebug committed:
Bump nightly release to fix wasm test error

cowboy-bebug committed:
Fix a doc error

cowboy-bebug committed:
Add scripts to support debugging

cowboy-bebug commented:
All the tests are now fixed 😄
Also added some useful scripts (https://github.com/plugblockchain/plug-blockchain/pull/117/commits/6fb409dda42114ef13681cc79edb82da71030de3) for similar debugging tasks in future.

cowboy-bebug committed:
Set -euo pipefail for best practice

jordy25519 commented:
fixes lvgtm.
Those bash scripts are 🔥 (did you chmod+x them?)

cowboy-bebug commented:
Yep all the executables are `x` (`logger.sh` is not):
```
-rwxr-xr-x   1 eric  staff  1495 27 Jul 08:48 bootstrap
-rw-r--r--   1 eric  staff   441 27 Jul 08:49 logger.sh
-rwxr-xr-x   1 eric  staff   703 27 Jul 08:49 overwrite-ui
-rwxr-xr-x   1 eric  staff  2018 27 Jul 08:49 test-wasm
```
